src/base.js

```diff
@@ -290,15 +302,22 @@ export default class KintoClientBase {
       const chunks = partition(requests, maxRequests);
       return pMap(chunks, chunk => this._batchRequests(chunk, options));
     }
-    const { responses } = await this.execute({
-      ...reqOptions,
-      path: endpoint("batch"),
-      method: "POST",
-      body: {
-        defaults: { headers },
-        requests: requests,
+    const { responses } = await this.execute(
+      {
+        // FIXME: is this really necessary, since it's also present in
+        // the "defaults"?
+        headers: headers,
```

Comment:
There actually are (unit) tests that ensure the current behavior, which is why I left the FIXME in.